backend/requirements/development.txt

```diff
@@ -0,0 +1 @@
+-r base.txt
```

Comment:
Looks like you've forgot to add current requirements to `base.txt`.

P.S. Cool trick with `-r base.txt`, I wasn't aware of it, thanks!